reflex/utils/prerequisites.py

```diff
@@ -824,11 +824,15 @@ def install_frontend_packages(packages: set[str], config: Config):
     # unsupported archs will use npm anyway. so we dont have to run npm twice
     fallback_command = (
         get_package_manager()
-        if constants.IS_WINDOWS and constants.IS_WINDOWS_BUN_SUPPORTED_MACHINE
+        if not constants.IS_WINDOWS
```

Comment:
Shouldn't the fallback always be `npm` regardless of OS based on what we discussed? Why is there a case where it's still None ?